static/styles.css

```diff
@@ -85,3 +85,303 @@ Kanban boards styles
   margin-bottom: 4px;
   font-size: 14px;
 }
+
+/* ========================================
```

Comment:
There is a lot of changes to the style here - should this all be underneath the edit-email-template-page so it doesn't impact other pages? Or have you checked the other pages and it is OK?

This seems to have a big affect on the whole application, not just email templates page?